openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -66,25 +67,26 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                 ${HTML(outline_fragment.body_html())}
             </main>
             <aside class="course-sidebar layout-col layout-col-a">
-                <div class="section section-tools">
-                    <h3 class="hd-6">${_("Course Tools")}</h3>
-                    <ul class="list-unstyled">
-                        <li>
-                            <a href="${reverse('openedx.course_bookmarks.home', args=[course_key])}">
-                                <span class="icon fa fa-bookmark" aria-hidden="true"></span>
-                                ${_("Bookmarks")}
-                            </a>
-                        </li>
-                        % if UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
-                            <li>
-                                <a href="${reverse('openedx.course_experience.course_updates', args=[course.id])}">
-                                    <span class="icon fa fa-newspaper-o" aria-hidden="true"></span>
-                                    ${_("Updates")}
-                                </a>
-                            </li>
-                        % endif
-                    </ul>
-                </div>
+                <%
+                course_tools = CourseToolsPluginManager.get_course_tools()
+                %>
+                % if course_tools:
```

Comment:
I should have mentioned this earlier, but this probably warrants a quick unit test with and without tools.  Or, if we already have coverage for with tools in other places (the tools themselves), maybe just test with none and comment that coverage for the other case is being handled elsewhere.